src/Settings/ElevarSettings.php

```diff
@@ -21,8 +27,34 @@ public function description(): string
         return 'Interact with Elevar.';
     }
 
+    public function sections()
+    {
+        return [
+           [
+               'title' => 'Key',
+               'properties' => 'uuid',
+           ],
+           [
+               'title' => 'Data Layer Listener',
+               'properties' => 'data_layer_listener_enabled',
+           ],
+           [
+               'title' => 'Server Side',
+               'properties' => 'server_side_url',
+           ],
+        ];
+    }
```

Comment:
All of these settings are all only really related to the Data Layer portion of things, so maybe just group them as one?

I've also added a new feature to Strata to allow to specify custom labels for each field. If not defined it will default back to current label. (Just remember to pull in `develop` first and run `composer install`.

```suggestion
    public function sections()
    {
        return [
           [
               'title' => 'Data Layer',
               'properties' => ['data_layer_listener_enabled', 'uuid', 'server_side_url'],
           ],
        ];
    }
    
    protected function labels() : array
    {
        return [
            'data_layer_listener_enabled' => 'Enable Elevar Data Layer',
            'uuid' => 'Key (UUID)',
            'server_side_url' => 'Server side URL',
        ];
    }
```